lib/bill_forward/entities/account.rb

```diff
@@ -7,6 +7,22 @@ module BillForward
   class Account < MutableEntity
   	@resource_path = BillForward::ResourcePath.new("accounts", "account")
 
+    class << self
+      def credit(id, query_object = {}, custom_client = nil)
```

Comment:
> As far as I can see every method beside `generic_entity` (and all the chain down through `MutableEntity`) has only static methods. I thought it was a design choice.

There are plenty of examples of dynamic methods upon billing entities in this repository.

```ruby
subscription.add_payment_method
subscription.remove_payment_method
subscription.get_payment_methods
subscription.migrate_plan
MutableEntity.save
MutableEntity.delete
```

Static methods _are_ useful, because they ensure you don't have to construct an instance of the object (like this):

```ruby
# option 1: fetch from API
acc = BillForward::Account::get_by_id 'ACC-123-ABC'

# option 2: construct a "sham"
acc = BillForward::Account.new({
  id => 'ACC-123-ABC'
})
```

But — at the same time — if you _do_ have an instance of the object, then:

```ruby
# this:
acc.credit

# is easier than this:
BillForward::Account.credit 'ACC-123-ABC'
```

So, there are advantages to both solutions.

If it's easy enough: I would recommend adding a method that invokes the static function. So it would be a "shortcut", which helps people who _already have an instance of that account_.